pynestml/codegeneration/resources_nest/point_neuron/setup/@MODULE_NAME@.cpp.jinja2

```diff
@@ -122,12 +122,17 @@ const std::string
 void
 {{moduleName}}::init( SLIInterpreter* i )
 {
-  {% for neuron in neurons %}
+{%- if neurons %}
```

Comment:
Is the extra `if` condition necessary? In case of no `neurons` or `synapses`, won't the list be empty?